### PR TITLE
Extends hubspot3 errors

### PR DESCRIPTION
**Pull request by jisson**
Extends the hubspot3 errors allowing user to import errors from `hubspot3` directly from `djhubspot`.

### CODE REVIEW

File: djhubspot/errors.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+    Provide an interface allowing users to import hubspot3 in a more intuitive way, directly from
+    djhubspot.
+    """
+    pass
+
+
+class DJHubspotError(Exception):
+    """
+    Sub class from hubspot3.
```
Comment (Jenselme) [inline djhubspot/errors.py]: It doesn't subclass from hubspot3 ;-)

File: djhubspot/errors.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+    Provide an interface allowing users to import hubspot3 in a more intuitive way, directly from
+    djhubspot.
+    """
+    pass
+
+
+class HubspotEventError(DJHubspotError):
+    """
+    Sub class from hubspot3.
```
Comment (Jenselme) [inline djhubspot/errors.py]: Same here.

File: djhubspot/errors.py (hunk 1 above)
Comment (jisson) [reply djhubspot/errors.py]: Ooops, thanks :)

File: djhubspot/errors.py (hunk 2 above)
Comment (jisson) [reply djhubspot/errors.py]: :+1:

**Review by Jenselme — APPROVED** [review]

**Pull request merged**